src/common/utils/router-context/index.ts

```diff
@@ -15,10 +15,11 @@ import generatePath from '../generate-path';
 export const createRouterContext = (
```

Comment:
Should we change the API of this function? It already has 4 arguments and might have more later. 
I think it might be better to change it to:

```js
createRouterContext({ route, params, query, basePath })
// or
createRouterContext(route, { params, query, basePath })
```


Before it starts to get used (AFAIK no one is using it yet)

Reply:
I was hesitant to introduce a breaking change but`createRouterContext(route, { params, query, basePath })` seems like a good idea, especially if no one's using it yet. I'll push in the change.

Reply:
Should we also introduce a similar signature for `findRouterContext`? `findRouterContext(routes, { location, basePath })`